package-build.el

```diff
@@ -1096,7 +1119,7 @@ and a cl struct in Emacs HEAD.  This wrapper normalises the results."
       (package-build--message "Built in %.3fs, finished at %s"
                   (time-to-seconds (time-since start-time))
                   (current-time-string))
-      file-name)))
+      (list file-name version))))
```

Comment:
This change is only for the tests, so if those are rejected this should be too (which is why they're in the same commit).